hr_payslip_ytd_amount/__openerp__.py

```diff
@@ -34,6 +34,11 @@
 The purpose of this module is to eliminate redondant rules that calculate
 the year-to-date value of other rules.
 
```

Comment:
Could you remove the description from the file, will be generated from readme file